lib/dialects/mssql/query/compiler.js

```diff
@@ -41,8 +41,9 @@ class QueryCompiler_MSSQL extends QueryCompiler {
   // inserts using a single query statement.
   insert() {
     const insertValues = this.single.insert || [];
-    let sql = this.with() + `insert into ${this.tableName} `;
     const { returning } = this.single;
+    let sql = this.with() + `${returning ? `declare @out table (${returning
```

Comment:
this is quite difficult to read. Could you extract whole new ternary block into a separate variable?

Reply:
Agreed. Simplifying and condensing repeated code for readability and maintainability.